bluemira/builders/EUDEMO/maintenance.py

```diff
@@ -238,6 +238,22 @@ def segment_blanket_xz(breeding_blanket_xz, r_inner_cut, cut_angle, cut_thicknes
 def build_upper_port_zone(r_up_inner, r_up_outer, z_max=10, z_min=0):
     """
     Make the void geometry for the upper port in the poloidal plane.
+
+    Parameters
+    ----------
+    r_up_inner: radius
```

Comment:
Is `radius` a type, or should this be `float`? 

Happy for this to be merged once this is fixed